notebooks/metatranscriptomics_pipeline_iSeqnorm.ipynb

```diff
@@ -0,0 +1,1477 @@
+   "source": [
+    "# get DNA & RNA concentratin information\n",
+    "dna_concs = make_2D_array(plate_df, data_col='Sample DNA Concentration', well_col=well_col).astype(float)\n",
+    "rna_concs = make_2D_array(plate_df, data_col='Sample RNA Concentration', well_col=well_col).astype(float)\n",
+    "\n",
+    "# get information for annotation\n",
+    "names = make_2D_array(plate_df, data_col='Sample', well_col=well_col)\n",
+    "\n",
+    "plot_plate_vals(dna_concs,\n",
+    "                annot_str=names,\n",
+    "                color_map='viridis',\n",
+    "                annot_fmt='.5s')\n",
+    "plt.title(\"DNA Concentrations\",fontsize=16)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Visualize plate RNA concentrations and plate map:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_plate_vals(rna_concs,\n",
+    "                annot_str=names,\n",
+    "                color_map='viridis',\n",
+    "                annot_fmt='.5s')\n",
+    "plt.title(\"RNA Concentrations\",fontsize=16)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sns.boxplot(x='Project Plate',y='Sample DNA Concentration',data=plate_df,fliersize=0)\n",
+    "sns.stripplot(x='Project Plate',y='Sample DNA Concentration',data=plate_df,color='grey',alpha=0.5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for plate in plate_df['Project Plate'].unique():\n",
+    "    plate_needs_dilution = requires_dilution(plate_df.loc[plate_df['Project Plate']==plate,\n",
+    "                          ['Sample DNA Concentration']], threshold=40)\n",
+    "    if plate_needs_dilution:\n",
+    "        #Dilution, 1:dilution factor\n",
+    "        #sample : total volume (as a factor)\n",
+    "        for dilution_factor in range(2,6):\n",
+    "            dilute_ = requires_dilution(plate_df.loc[plate_df['Project Plate']==plate,\n",
+    "                              ['Sample DNA Concentration']]/dilution_factor, threshold=40)\n",
+    "            if dilute_:\n",
+    "                continue\n",
+    "            else:\n",
+    "                print(f'You  need to do a a 1:{dilution_factor} dilution for {plate}')\n",
+    "                break\n",
+    "    else:\n",
+    "        print(f'You do not need a dilution for {plate}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Post DNAse QC (Optional but highly encouraged)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dnase_dna_concs_fp = './test_data/Quant/MiniPico/2024_NPH_Plates_3-6_Post_DNase_gDNA_Quant.txt'\n",
+    "\n",
+    "if not os.path.isfile(sample_dna_concs_fp):\n",
```

Comment:
this is checking if `sample_dna_concs_fp` already exists, not if `dnase_dna_concs_fp` already exists; I think it should be checking the latter.

Reply:
you are correct, thanks for catching this. Will fix it